Chemistry/src/DataModel/ConfigurationFile.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+
+using Microsoft.Quantum.Simulation.Core;
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.Quantum.Chemistry;
+
+namespace Microsoft.Quantum.Chemistry
+{
+
+    
+    /// <summary>
+    /// Configuration settings for modifying chemistry library behavior.
+    /// </summary>
+    public class Config
+    {
+
+        /// <summary>
+        /// Construct default configuration.
+        /// </summary>
+        /// <returns>Default configuration class.</returns>
+        public static Config Default()
+        {
+            return new Config();
+        }
+
+        /// <summary>
+        /// Default configuration constructor;
+        /// </summary>
+        public Config()
+        {
+            indexConvention = IndexConvention.Default;
+            truncationThreshold = TruncationThreshold.Default;
+        }
+
+        /// <summary>
+        /// Choose indexing convention from spin-orbital index to an integer.
+        /// </summary>
+        public IndexConvention.Type indexConvention;
```

Comment:
In general, public properties should start with initial capital letters.
```suggestion
        public IndexConvention.Type IndexConvention;
```

Reply:
Noted

Reply:
Fixed